### PR TITLE
FYLE-2891ub0

### PR DESCRIPTION
**Pull request by Kulsekar4**

### CODE REVIEW

**Review by aditya999123 — APPROVED** [review]

File: src/components/schemas/statements.yaml [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+              Id of the existing duplicate statement
+            example:
+              - stmpge49ielgel
+            nullable: true
+    unique_id:
+      type: string
+      description: |
+        Unique hash value that is used to identify the statement.
+      example: 74175aadf22675aa59df28b96d351cfb
+      nullable: false
```
Comment (modernwarfareuplink) [inline src/components/schemas/statements.yaml:109]: nullable: false

is default so not needed

File: src/components/schemas/statements.yaml [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+      nullable: true
+    end_date:
+      $ref: './fields.yaml#/timestamptz_utc'
+      description: |
+        End date of the statement period
+      nullable: true
+    has_duplicate_statements:
+      type: boolean
+      description: |
+        This specifies if the statement is completely duplicate or not.
```
Comment (modernwarfareuplink) [inline src/components/schemas/statements.yaml:67]: p1 | This specifies if the statement is complete duplicate of already uploaded statement or not.

**Review by modernwarfareuplink — APPROVED** [review]
lgtm, added minor p1/p2 comments

**Review by modernwarfareuplink — APPROVED** [review]
lgtm

**Pull request closed without merging**